extensions/amp-experiment/amp-experiment.md

```diff
@@ -16,4 +16,90 @@ limitations under the License.
+
+## Behavior
+The `<amp-experiment>` element is used to conduct user experience experiments on an AMP document. It provides hooks to define customizable variants and allocates traffic to each of the variants based on the configuration. For each page view, the variant allocation is also exposed to `amp-pixel` and `amp-analytics` so that the necessary data can be collected to perform statistical comparison across variants.
+
+A user-sticky variant assignment is supported to provide a consistent user experience to the same client. This functionality relies upon AMP’s [Client ID](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#client_id) capability to provide random values that are consistent across page views. Please be aware that usage of this feature with this behavior might require updating your privacy policy, or obtaining end user consent in some jurisdictions. (If this is relevant for you, please see `consentNotificationId` below.)
+
+Multiple experiments can be run on the same AMP document in parallel with their own sets of variants. In user sticky mode, the allocations are orthogonal among different experiments, meaning there will be no correlation between 2 variants (user groups) that are from different experiments.
+
+## Configuration
+The configuration of the experiments is specified in a JSON object. 
+```
+<amp-experiment>
+  <script type=”application/json”>
+    {
+      aExperiment: {
+        sticky: true, 
+        consentNotificationId: “consent-notif”,
+        variants: {
+          treatment1: 12.5,
+          treatment2: 12.5,
+          treatment3: 25.0,
+        },
+      },
+      bExperiment: {...}
+    }
+  </script>
+</amp-experiment>
+```
+
+At top level, the JSON is a map of experiment configurations keyed by experiment names. In each experiment, available settings are described in the table below:
+
+| Name                  | Is required field?           | Description |
+|-----------------------|------------------------------|-------------|
+|`sticky`               | No, default=`true`           | Whether the experiment is user sticky or not. |
+|`consentNotificationId`| No, default=`undefined`      | The element ID of the `amp-user-notification` to be dismissed before a user sticky can be conducted. To not block the page rendering, an experiment with this field specified will be skipped if the consent is not provided prior to the current visit. That’s to say, only returning visits with user consent can trigger such an experiment. It only works when `sticky=true`. |
+|`variants`             | Yes                          | A name-to-percentage map where percentage is a float number in range (0, 100) that indicates the amount of traffic will be allocated to the variant. Variants don’t have to sum up to 100%. In that case, there’ll be a portion of the traffic allocated to none, which is a reserved keyword that indicates “no variant allocated”. |
```

Comment:
allocated to a variant named "none", which is a reserved keyword that indicates no variant was allocated.